web-app/src/routes/register.svelte

```diff
@@ -0,0 +1,99 @@
+<script>
+  import { goto } from "@sapper/app";
+
+  let name = '';
+  let phone = '';
+  let league = '';
+  let tags = [];
+
+  function submit() {
+    console.log('Registering', { name, phone, league, tags });
+
+    goto('confirm-code');
+  }
+</script>
+
+<style>
+  form {
+    display: grid;
+    grid-template:
+      "label-name  input-name " auto
+      "label-phone input-phone" auto
+      "league      league-opts" auto
+      "tags        tags       " auto
+      "submit      submit     " auto /
+       1fr        1fr;
```

Comment:
This is so fucking cool...

Reply:
😀 this is why I was glad we weren't build for antique browsers